### PR TITLE
Classification model changes

### PR DESCRIPTION
**Pull request by subutai**
Defined trainText and classifyText in ClassificationModel.
Implemented trainText and classifyText for HTM and keywords model.
Added LanguageSensor->KNN model configuration.

Added simple hello_classification_model script for running models. This can eventually be converted into a simple runner and into tests of the system.

Fixed type bugs in TemporalPoolerRegion.py and LanguageSensor.py

### CODE REVIEW

**Comment by subutai** [comment]
@BoltzmannBrain

File: htmresearch/frameworks/nlp/classify_keywords.py [hunk 1]
```diff
@@ -44,23 +44,42 @@ def __init__(self,
...
 
+  def encodeToken(self, token):
+    """
+    Randomly encode an SDR of the input token. We seed the random number
+    generator such that a given string will yield the same SDR each time this
+    method is called.
+
+    @param token  (str)      String token
+    @return       (list)     Numpy arrays, each with a bitmap of the
+                                        encoding.
```
Comment (BoltzmannBrain) [inline htmresearch/frameworks/nlp/classify_keywords.py]: spacing is off

**Comment by BoltzmannBrain** [comment]
:+1:

**Comment by BoltzmannBrain** [comment]
@subutai the inputs to nupic regions (i.e. KNN) need to be changed to match the output data types of this PR.

**Comment by subutai** [comment]
> @subutai the inputs to nupic regions (i.e. KNN) need to be changed to match the output data types of this PR.

I don't think so. I changed these data types so that we match the NuPIC standard!

**Pull request merged**